docs/api/api_blueprint.apib

```diff
@@ -4142,6 +4142,424 @@ Data related to the various tracks (e.g, python, AI, etc.) associated with the v
+                        "facebook": null,
+                        "gender": "Male",
+                        "position": null,
+                        "organisation": null,
+                        "sponsorship-required": "",
+                        "heard-from": null
+                    },
+                    "type": "speaker",
+                    "id": "3",
+                    "links": {
+                        "self": "/v1/speakers/3"
+                    }
+                },
+                {
+                    "relationships": {
+                        "user": {
+                            "links": {
+                                "self": "/v1/speakers/1/relationships/user",
+                                "related": "/v1/speakers/1/user"
+                            }
+                        },
+                        "event": {
+                            "links": {
+                                "self": "/v1/speakers/1/relationships/event",
+                                "related": "/v1/speakers/1/event"
+                            }
+                        },
+                        "sessions": {
+                            "links": {
+                                "self": "/v1/speakers/1/relationships/session",
+                                "related": "/v1/speakers/1/sessions"
+                            }
+                        }
+                    },
+                    "attributes": {
+                        "website": "https://firstSpeaker-Patched.com",
+                        "city": "Delhi",
+                        "short-biography": "",
+                        "name": "firstSpeaker-Patched",
+                        "speaking-experience": "",
+                        "country": "India",
+                        "twitter": null,
+                        "is-featured": false,
+                        "linkedin": null,
+                        "email": "firstSpeaker_Patched@gmail.com",
+                        "long-biography": "",
+                        "mobile": null,
+                        "github": null,
+                        "facebook": null,
+                        "gender": "Male",
+                        "position": null,
+                        "organisation": null,
+                        "sponsorship-required": "",
+                        "heard-from": null
+                    },
+                    "type": "speaker",
+                    "id": "1",
+                    "links": {
+                        "self": "/v1/speakers/1"
+                    }
+                }
+            ],
+            "jsonapi": {
+                "version": "1.0"
+            },
+            "links": {
+                "self": "/v1/events/1/speakers"
+            }
+        }
+
+### Create Speaker [POST]
+
++ name (string) - The name of the speaker **(required)**
++ email (string) - The email of the speaker **(required)**
++ photo_url (string) - URL of the photo of the speaker
```

Comment:
Fields like this and other are all dasherized. Please change them to table format as well. @niranjan94 mentioned it here https://github.com/fossasia/open-event-orga-server/issues/3822